content/en/service_management/app_builder/connections/http_request.md

```diff
@@ -143,6 +147,29 @@ You can also select whether you want to allow expired certificates, or follow re
 
 Under **Error on Status**, enter a comma-delineated list of any status codes on which to return an error. Use the **Response Parsing** dropdown to override the default response parsing method inferred from the headers, and **Response Encoding** if the target server specifies the wrong encoding in its response headers.
 
+## Private actions
+
+{{< callout url="https://google.com" btn_hidden="false" header="Join the Beta!">}}
+Private Actions are in beta. Use this form to request access today.
+{{< /callout >}}
+
+You can use a private HTTP action to interact with services hosted on your private network without opening a port in your firewall. Private actions make use of a private action runner which you install on a host in your network and pair with a Datadog Connection. For more information, see [Private Actions][5].
```

Comment:
copy nit: "without opening a port in your firewall" -> "without exposing your services to the public internet"

there's a few places where this copy is found so they should all be updated